data_validation/query_builder/random_row_builder.py

```diff
@@ -100,7 +103,13 @@ def compile(
             table_name (String): The name of the table to query.
         """
         table = clients.get_ibis_table(data_client, schema_name, table_name)
-        randomly_sorted_table = self.maybe_add_random_sort(data_client, table)
+        # Build Query Expressions
+        calc_table = table
```

Comment:
This line looks unnecessary, we can use table in the places we use calctable

Reply:
Modified